scripts/aws/cfn/storage.yml

```diff
@@ -76,6 +91,12 @@ Resources:
             Prefix: indexes/
             NoncurrentVersionExpiration:
               NoncurrentDays: !Ref IndexesBucketVersionExpiresAfterDays
+          - Id: DeleteOldVersionsForRaddStoreRegistry
```

Comment:
Cambierei il nome del bucket in `!Sub ${ProjectName}-bff-data-${AWS::Region}-${AWS::AccountId}` e il `Prefix: indexes/` in `Prefix: sender-dashboard/indexes/`. Se viene fatta questa modifica, va cambiato il valore di PnIndexObjectKey in microservice-dev-cfg.json